Blish HUD/GameServices/ArcDps/Models/Ag.cs

```diff
@@ -5,11 +5,29 @@
     /// </summary>
     public class Ag {
 
+        /// <summary>
+        /// Agent name. May be null. Valid only at time of event.
```

Comment:
Due to us copying the data over, the agent name is valid for as long as you need it. I would suggest removing the last sentence.